core/test/Lender.t.sol

```diff
@@ -91,7 +98,7 @@ contract LenderTest is Test {
             18
         );
         // Make sure `borrowIndex` is just as precise as `accrualFactor`
```

Comment:
`accuralFactor` should be `yieldPerSecond`?

Reply:
Yes that, and also I need to fix this as `yieldPerSecond` is _never_ greater than `1e12`

Good catch